aten/src/THC/generic/THCStorage.h

```diff
@@ -6,7 +6,8 @@
 #define TH_STORAGE_RESIZABLE  2
 #define TH_STORAGE_FREEMEM    4
 
-typedef struct THCStorage THCStorage;
+// Blah.  I tried to do this with a typedef but it didn't work
```

Comment:
I think this is what you want anyway -- that's how all our renames are defined, and you really want substitution.